### DISCUSSION TITLE
Noob Question regarding setting up a nebula network

### DISCUSSION THREAD

**Discussion by tliu0c** [slackhq/nebula#1061] [Q&A] [answered]
Hi there, this might be a bit of a noob question. But what I am trying to do is setup a nebula network consisting a couple devices in different locations. They will have private IPs in the nebula network and can communicate with each other, ie, http, ssh, smb, etc.

I have created such a network using a oracle cloud free tier VPS as lighthouse, and the devices can ping each other fine with the private IPs. But the problem is that ONLY ping works. The moment I start using other services like http then nothing it getting through. For example, If I curl the nebula IP address on the http server itself, the webpage shows fine. But on other devices, it doesn't work. Almost like some kind of firewall is blocking it. What am I missing here? Thanks in advance!

**Comment by johnmaguire** [comment, marked as answer]
Hi @tliu0c, this is probably a silly question, but have you configured the [Nebula firewall](https://nebula.defined.net/docs/config/firewall/) appropriately for the services you're trying to use? Can you share your `firewall` section?

**Reply by tliu0c** [reply]
Thanks John, that is the issue! It works now.
